cvmfs/cvmfs_server

```diff
@@ -484,19 +484,22 @@ check_multiple_repository_existence() {
 
 
 # checks if the aufs kernel module is present
-#
+# of if aufs is compiled in
 # @return   0 if the aufs kernel module is loaded
 check_aufs() {
-  /sbin/modprobe -q aufs || test -d /sys/fs/aufs
+  /sbin/modprobe -q aufs || test -d /sys/fs/aufs || \
+  test x"$(zgrep AUFS_FS /proc/config.gz | awk -F = '{print $2}')" = xy	
```

Comment:
Shouldn't this case be already covered by `test -d /sys/fs/aufs` in the line above?